lib/activerecord-import/import.rb

```diff
@@ -335,18 +335,16 @@ def import_helper( *args )
         end
 
         array_of_attributes = models.map do |model|
-            # this next line breaks sqlite.so with a segmentation fault
-            # if model.new_record? || options[:on_duplicate_key_update]
-            column_names.map do |name|
-              name = name.to_s
-              if respond_to?(:defined_enums) && defined_enums.key?(name) # ActiveRecord 5
-                model.read_attribute(name)
-              elsif model.class.column_defaults[name].is_a?(Integer)
```

Comment:
I think I let this change slip through without a test in 511533450b69c0755fae0f2faf493a7d7356c3d1 . If you're able to add a test for this @jkowens that would be great. If not let me know and I'll put it on my docket. I want to make sure we don't back out an actual fix for a bug fix.